specification.rst

```diff
@@ -567,11 +567,9 @@ Terms may be defined for specific forms by using ``cs:term`` with the optional
 -  "verb-short" - e.g. "ed." for the term "editor"
 -  "symbol" - e.g. "§" and "§§" for the term "section"
 
-If a style uses a term in a form that is undefined (even after `Locale
-Fallback`_), there is fallback to other forms: "verb-short" first falls back to
-"verb", "symbol" first falls back to "short", and "verb" and "short" both fall
-back to "long". If no locale or form fallback is available, the term is rendered
-as an empty string.
+If a style uses a term in a form that is undefined (even after `Locale Fallback`_), there is fallback to other forms:
+"verb-short" first falls back to "verb", "symbol" first falls back to "short", and "verb" and "short" both fall back to "long". 
+In addition, the terms "long-ordinal-1" to "long-ordinal-10" fall back to the set of ordinal suffix terms. If no locale or form fallback is available, the term is rendered as an empty string.
```

Comment:
I think my question at the time was _What if 42 was asked to be rendered as a long ordinal?_. In this case would we simply match 42 against the set of ordinals? It seems this is what the change suggests processors should do (which makes sense to me).

It's been a long time since I've read that part of the spec; I'm not even sure whether what I'm saying make sense at all :-)

Reply:
That's already defined in the spec:
```
Long ordinals are defined with the terms “long-ordinal-01” to “long-ordinal-10”, which are used for the numbers 1 through 10. For other numbers “long-ordinal” falls back to “ordinal”.
```